src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/EditApprenticeship.cshtml

```diff
@@ -168,51 +169,50 @@
-                            <div class="govuk-radios">
-                                <div class="govuk-radios__item">
-                                    <input type="radio" asp-for="DeliveryModel" class="govuk-radios__input" value="@DeliveryModel.Regular" id="DeliveryModelNormal" aria-describedby="DeliveryModelNormal-hint">
-                                    <label for="DeliveryModelNormal" class="govuk-label govuk-radios__label">
-                                        Regular
-                                    </label>
-                                    <div id="DeliveryModelNormal-hint" class="govuk-hint govuk-radios__hint">
-                                        The apprentice will have a single employer
-                                    </div>
-                                </div>
-                                <div class="govuk-radios__item">
-                                    <input type="radio" asp-for="DeliveryModel" class="govuk-radios__input" value="@DeliveryModel.PortableFlexiJob" id="DeliveryModelFlexible" aria-describedby="DeliveryModelFlexible-hint">
-                                    <label for="DeliveryModelFlexible" class="govuk-label govuk-radios__label">
-                                        Portable flexi-job
-                                    </label>
-                                    <div id="DeliveryModelFlexible-hint" class="govuk-hint govuk-radios__hint">
-                                        The apprentice will move between multiple employers
-                                    </div>
-                                </div>
-                            </div>
-                        </fieldset>
+                    if (!string.IsNullOrWhiteSpace(Model.CourseCode))
+                    {
+                        Model.CourseName = (await CommitmentsApiClient.GetTrainingProgramme(Model.CourseCode)).TrainingProgramme.Name;
+                    }
+
+                    if (@Model.DeliveryModel != DeliveryModel.Regular)
+                    {
+                        <div class="govuk-form-group">
+                            <p class="govuk-!-font-weight-bold govuk-!-margin-bottom-0">Delivery model</p>
+                            <p class="govuk-body">
+                                @Model.DeliveryModel.ToDescription()
+                                <button type="submit" name="ChangeDeliveryModel" value="Edit" class="das-button--inline-link">
+                                    Edit<span class="govuk-visually-hidden"> delivery model</span>
+                                </button>
+                            </p>
+                        </div>
+                    }
+
+                    <div class="govuk-form-group">
+                        <p class="govuk-!-font-weight-bold govuk-!-margin-bottom-0">Apprenticeship training course</p>
```

Comment:
I think the apprenticeship should come above the delivery model, to match the order they are seen in the workflow.

Reply:
Agree